### ISSUE TITLE
Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by mxstbr** [feross/thanks#4]
```
$ npx thanks
Error: Cannot read property 'map' of undefined
```

This line:

https://github.com/feross/thanks/blob/1116cad642764d2f2852503f4f54a06b46c733ba/cmd.js#L200

**Comment by feross** [comment]
Thanks for reporting. Confirmed. Working on it.

**Comment by feross** [comment]
Fixed and released a new version 0.3.1.

**Comment by mxstbr** [comment]
💜
